crates/lune-roblox/src/instance/mod.rs

```diff
@@ -337,14 +337,14 @@ impl Instance {
     /**
         Gets a property for the instance, if it exists.
     */
-    pub fn get_property(&self, name: impl AsRef<str>) -> Option<DomValue> {
+    pub fn get_property(&self, name: &str) -> Option<DomValue> {
         INTERNAL_DOM
             .lock()
             .expect("Failed to lock document")
             .get_by_ref(self.dom_ref)
             .expect("Failed to find instance in document")
             .properties
-            .get(name.as_ref())
+            .get(&name.into())
```

Comment:
If we don't want to change the type of `name` param to `Ustr`, then we can leave this method signature alone, and do `&ustr(name.as_ref())` for the property get